SocialFilter/sections/specific.txt

```diff
@@ -51,6 +51,7 @@ stardoll.com,yakugakulab.info,colopl.co.jp,carnegie.ru,shamanking-project.com,ya
 !
 bellemaison.jp##.external-buttons
 united-arrows.co.jp##.mlShare
+kikkoman.co.jp##.share-container-tsushin
```

Comment:
Hmm, I can't find this. Do you have URL? I found `##.share--content` on emulated Android but this hides print button too (extended syntax is required to avoid).

Reply:
This page have that element.
https://www.kikkoman.co.jp/homecook/tsushin/index.html
![](https://user-images.githubusercontent.com/47618415/151789233-3ec22c32-7a2d-4017-b970-48ba4d152ce5.png)

Reply:
Thanks!

Reply:
Okay, actually extended syntax was not needed. Add `kikkoman.co.jp##.share--content > ul > li > button[onclick$="ShareBtn\(\);"]` found on `https://www.kikkoman.co.jp/homecook/search/recipe/00051984/index.html`.

Reply:
np

Reply:
> Okay, actually extended syntax was not needed. Add kikkoman.co.jp##.share--content > ul > li > button[onclick$="ShareBtn\(\);"] found on https://www.kikkoman.co.jp/homecook/search/recipe/00051984/index.html.
 

done